.github/ISSUE_TEMPLATE/new_issue_template.yml

```diff
@@ -0,0 +1,63 @@
+name: SS3 Team Issues - New Template
+description: Issue template for SS3 team members ONLY
+title: "[Issue]: "
+assignees:
+  - Rick-Methot-NOAA
+body:
+- type: markdown
+  attributes:
+    value: |
+      Thanks for taking the time to fill out this issue!
+- type: textarea
+  id: describe
+  attributes:
+    label: Issue Description
+    description: Please provide a description of the issue.
+  validations:
+    required: true
+- type: textarea
+  id: summary
+  attributes:
+    label: Summary of Issue History, Discussion, and Major Aspects of Code Development 
+    description: A summary of pertinent information on this issue, including major aspects of code development, that occurred in the discussion/via email. The person creating the issue is responsible for adding info from the discussion to this section.
```

Comment:
Do you want to include something about how this section can be added on to/edited at any time in the future and should be summarized prior to closing? Perhaps in the second sentence?

Reply:
Reworded second sentence to include your suggested text in new commit.